upf/test/test_upf.py

```diff
@@ -1048,6 +1061,81 @@ def verify_gtp_no_echo(self, to_send):
         self.send_from_grx(to_send)
         self.if_grx.assert_nothing_captured()
 
+    def send_from_sgi_to_ue(self, payload=None, l4proto=UDP, ue_port=12345,
```

Comment:
I believe this method only differs `PFCPHelper.send_from_sgi_to_access()` by its name? Then, I guess the right thing to do is to just rename `PFCPHelper.send_from_sgi_to_access()` to `PFCPHelper.send_from_sgi_to_ue()` or `PFCPHelper.send_from_sgi()` as basically they do the same thing